Apps/WebClient/src/ClientApp/src/utility/errorTranslator.ts

```diff
@@ -24,13 +25,15 @@ export default class ErrorTranslator {
                 description: this.getDisplayMessage(error.errorCode),
                 detail: error.resultMessage,
                 errorCode: error.errorCode,
+                supportDetails: `If this issue persists, contact HealthGateway@gov.bc.ca and provide ${error.traceId}.`,
```

Comment:
I think this doesn't belong here since the BannerError only contains the information for display.  The error card is the actual piece that is being rendered, so it should have the message using the traceId.
Also, if there is no TraceId, the message would not conclude correctly